src/Cake.Common/Tools/ReportGenerator/ReportGeneratorReportType.cs

```diff
@@ -14,25 +14,60 @@ public enum ReportGeneratorReportType
         /// </summary>
         Badges = 1,
 
+        /// <summary>
+        /// Cobertura report
+        /// </summary>
+        Cobertura,
+
+        /// <summary>
+        /// CSV summary report
+        /// </summary>
+        CsvSummary,
+
         /// <summary>
         /// Html report.
         /// </summary>
-        Html = 2,
+        Html,
```

Comment:
You will notice that I removed the explicit enum numbers.

Would you like me to:
1. either, add them explicitly in alphabetical order
2. or, add the new ad the bottom and preserve the original numbering
3. or, keep as it is now?

Second option probably doesn't make sense as I don't think that any external code would explicitly rely on those specific values, but you never know...

Reply:
Since this is a public enum, anything but option 2 is a breaking change. Hence, I’d go with that option.